herbert_core/utilities/classes/@object_lookup/rand_ind.m

```diff
@@ -118,6 +118,10 @@
 %              ind (and ielmts if present), and the inner dimensions as those
 %              of a parameter for a particular value of ind (and ielmts)
 %
+%               In most cases this means that for any given list of arguments
+%              each member is "local" and each member of the list corresponds to each element
```

Comment:
Going to be honest and say this is very opaque to me. But I wouldn't want to stop your work if you feel this improves things. Just asking that you terminate the sentence to make parsing a bit more obvious.:

```suggestion
%              each member is "local" and each member of the list corresponds to each element.
```

Consider this approval after the full stop gets in.